### ISSUE TITLE
How to set App Store user as Sandbox User

### ISSUE THREAD

**Issue by masarapmabuhay** [chrisamanse/LIKHA-Guide#18]
Salut! In Step 4 of `Part 3: In-App Purchases`, the LIKHA-Guide says, 

> "While in the device, make sure the App Store user is the Sandbox User you created."

May I know how this is done?

I've already created a Sandbox User on iTunes Connect. I've also downloaded the app on an actual device, but I'm not sure how to set my `App Store user` to my `Sandbox User`.

I've checked the [iTunes Connect Developer Guide](https://developer.apple.com/library/content/documentation/LanguagesUtilities/Conceptual/iTunesConnect_Guide/Chapters/SettingUpUserAccounts.html), but to no avail. 

Does this mean that I have to run the app on a real device via Xcode?

Merci beaucoup!

**Comment by chrisamanse** [comment]
Yes, testing in-app purchases has to be done on a real iOS device. Changing the App Store user can be done in Settings of the iOS device.

**Comment by masarapmabuhay** [comment]
Merci beaucoup pour ta réponse! Did you mean that I should change the `Apple ID` to my `Sandbox User ID` in the `iTunes Store and App Store` Settings? Merci encore une fois!

**Comment by chrisamanse** [comment]
Yes. Here's Apple's guide: https://developer.apple.com/library/content/documentation/LanguagesUtilities/Conceptual/iTunesConnectInAppPurchase_Guide/Chapters/TestingInAppPurchases.html

**Comment by masarapmabuhay** [comment]
Salut! Merci beaucoup pour ta réponse! I've read the `Apple guide` you sent, but I encountered an issue with the fact that the version of Xcode that I'm using based on the LIKHA-Guide is lower than the iOS version of my actual device. 

[Answers](http://stackoverflow.com/questions/33314733/could-not-find-developer-disk-image/33352278) from Stackoverflow say that the solution is to simply upgrade Xcode. However, as we've previously learned, new Warnings and Issues occur in the LIKHA repo when Xcode is upgraded.

In the meantime, I was looking at using `Promo Codes` to unlock languages via in-app purchasing, but I do not know where to actually put the `Promo Code`. When I choose "Buy", and press "Continue" to provide the `billing info`, there is no option to enter a `Promo Code` at all.

Merci toujours!
